pandas/core/missing.py

```diff
@@ -236,9 +236,17 @@ def _interpolate_scipy_wrapper(x, y, new_x, method, fill_value=None,
         alt_methods['pchip'] = interpolate.pchip_interpolate
     except AttributeError:
         if method == 'pchip':
-            raise ImportError("Your version of scipy does not support "
+            raise ImportError("Your version of Scipy does not support "
                               "PCHIP interpolation.")
 
+    try:
```

Comment:
can you change this (and the pchip one to something like):

```
if method == 'akima':
    alt_methods['akima'] = _akima_interpolate
```

Move to ``_akima_interpolate``
```
    from scipy.interpolate import Akima1DInterpolator  # noqa
```